README.md

```diff
@@ -2,608 +2,8 @@
 
```

Comment:
Should probably update the header to `Lattice Software Releases`

Reply:
[0a91ce8](https://github.com/GridPlus/lattice-software-releases/pull/4/commits/0a91ce832067e31755a54e4d539d149403afe9e4)